README.md

```diff
@@ -112,6 +112,7 @@ Content for taxon pages is returned by a search in Rummager based on content_ids
     - content tagged to a particular Topic, Mainstream browse page or Organisation
 - [email-alert-api](https://github.com/alphagov/email-alert-api), provides:
     - support for subscribing to notifications from a topic
+- [yarn][] - package manager for JavaScripts
```

Comment:
Should this link to [this](https://yarnpkg.com/)? Or even be a link?

Reply:
Ah, well it's using the reference style (with a [definition at the end of the document](https://github.com/alphagov/collections/pull/2070/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R156)). I often prefer this way as it's less verbose and if you need to update the link you only do it once per markdown page. It [renders fine](https://github.com/alphagov/collections/pull/2070/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5), but equally happy to make it a link.

Reply:
It wasn't working when I looked at it before but it looks fine now so I'll withdraw this!

Reply:
I initial forgot a space before the definitions at the end. So your comment helped me spot that!